From: Daniel Kahn Gillmor <dkg@fifthhorseman.net>
Date: Thu, 11 Jun 2015 18:26:16 -0400
Subject: Remove upstream git fanciness

Debian needs to be able to build libgpg-error while using git for
tracking the debian packaging itself.  Debian also wants to autoreconf
this stuff.  However, Debian does *not* need to conform exactly to
upstream's git configurations.

This avoids mistakes like accidentally identifying running code as a
beta.

This is the same approach taken in debian's libgpg-error packaging
since 2017.

Forwarded: not-needed
---
 autogen.sh | 76 ++++----------------------------------------------------------
 1 file changed, 4 insertions(+), 72 deletions(-)

--- a/autogen.sh
+++ b/autogen.sh
@@ -475,50 +475,6 @@ EOF
     die_p
 fi
 
-# Check the git setup.
-if [ -d .git ]; then
-  CP="cp -p"
-  # If we have a GNU cp we can add -v
-  if cp --version >/dev/null 2>/dev/null; then
-    [ -z "${SILENT}" ] && CP="$CP -v"
-  fi
-  if [ -f .git/hooks/pre-commit.sample ] \
-      && [ ! -f .git/hooks/pre-commit ]; then
-    [ -z "${SILENT}" ] && cat <<EOF
-*** Activating trailing whitespace git pre-commit hook. ***
-    For more information see this thread:
-      https://mail.gnome.org/archives/desktop-devel-list/2009-May/msg00084.html
-    To deactivate this pre-commit hook again move .git/hooks/pre-commit
-    and .git/hooks/pre-commit.sample out of the way.
-EOF
-      $CP .git/hooks/pre-commit.sample .git/hooks/pre-commit
-      chmod +x  .git/hooks/pre-commit
-  fi
-
-  if [ "$gettext_vers" != "n/a" ]; then
-    tmp=$(git config --get filter.cleanpo.clean)
-    if [ "$tmp" != \
-          "awk '/^\"POT-Creation-Date:/&&!s{s=1;next};!/^#: /{print}'" ]
-    then
-      info "*** Adding GIT filter.cleanpo.clean configuration."
-      git config --add filter.cleanpo.clean \
-        "awk '/^\"POT-Creation-Date:/&&!s{s=1;next};!/^#: /{print}'"
-    fi
-  fi
-  if [ -f build-aux/git-hooks/commit-msg ] \
-     && [ ! -f .git/hooks/commit-msg ]; then
-      [ -z "${SILENT}" ] && cat <<EOF
-*** Activating commit log message check hook. ***
-EOF
-      $CP build-aux/git-hooks/commit-msg .git/hooks/commit-msg
-      chmod +x  .git/hooks/commit-msg
-      if [ x"${display_name}" != x ]; then
-         git config format.subjectPrefix "PATCH ${display_name}"
-         git config sendemail.to "${patches_to}"
-      fi
-  fi
-fi
-
 aclocal_flags="-I m4"
 if [ -n "${extra_aclocal_flags}" ]; then
   aclocal_flags="${aclocal_flags} ${extra_aclocal_flags}"
